app/webpack/users/edit/components/content.jsx

```diff
@@ -0,0 +1,297 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { MenuItem, DropdownButton } from "react-bootstrap";
+
+import CheckboxRowContainer from "../containers/checkbox_row_container";
+import SettingsItem from "./settings_item";
+
+import PlaceAutocomplete from "../../../observations/identify/components/place_autocomplete";
+
+const radioButtons = {
+  any: I18n.t( "views.users.edit.project_addition_preferences.any" ),
+  joined: I18n.t( "views.users.edit.project_addition_preferences.joined" ),
+  none: I18n.t( "views.users.edit.project_addition_preferences.none" )
+};
+
+const obsFields = {
+  anyone: I18n.t( "anyone" ),
+  curators: I18n.t( "curators" ),
+  only_you: I18n.t( "only_you" )
+};
+
+const Content = ( {
+  profile,
+  handleInputChange,
+  handleCustomDropdownSelect,
+  handleDisplayNames,
+  handlePlaceAutocomplete,
+  showModal
+} ) => {
+  const createRadioButtons = ( ) => Object.keys( radioButtons ).map( button => (
+    <div key={button}>
+      <input
+        id="user_preferred_project_addition_by"
+        type="radio"
+        value={button}
+        name="preferred_project_addition_by"
+        checked={profile.preferred_project_addition_by === button}
+        onChange={handleInputChange}
+      />
+      <label className="margin-left" htmlFor="user_preferred_project_addition_by">
+        {radioButtons[button]}
+      </label>
```

Comment:
Currently if you click the label text, it doesn't change the setting, because each `<input>` in the radio group has the same ID. They should have unique IDs and each label's `for` attribute should reference the ID of the input it's labeling. However, you could just ignore the id attributes by making the input a child of the label.